### PR TITLE
Description added to generated Model code

### PR DESCRIPTION
**Pull request by andrioid**
I decided to take a stab at: https://github.com/99designs/gqlgen/issues/118

GraphQL descriptions are now written as comments on top of some of the Go types. I'm not sure if this covers all the basics. But it would be nice if someone could take a look and see if I'm on the right track.

### CODE REVIEW

**Comment by andrioid** [comment]
Sorry I broke the build. It seems there are some newlines that don't match the expected code generation.

I have never used Go templates for anything before, so I'm a bit lost in how to fix that.

File: codegen/templates/models.gotpl [hunk 1]
```diff
@@ -9,10 +9,13 @@ import (
 )
 
 {{ range $model := .Models }}
+	{{with .Description}} {{.|prefixLines "// "}} {{end}}
+
 	{{- if .IsInterface }}
 		type {{.GoType}} interface {}
 	{{- else }}
 		type {{.GoType}} struct {
+
```
Comment (vektah) [inline codegen/templates/models.gotpl]: Another extra newline

File: codegen/templates/models.gotpl [hunk 2]
```diff
@@ -9,10 +9,13 @@ import (
 )
 
 {{ range $model := .Models }}
+	{{with .Description}} {{.|prefixLines "// "}} {{end}}
+
```
Comment (vektah) [inline codegen/templates/models.gotpl]: One of the newlines are here (use `{{- with .Description}}`) to remove it.

File: codegen/templates/models.gotpl [hunk 3]
```diff
@@ -25,12 +28,13 @@ import (
 {{- end}}
 
 {{ range $enum := .Enums }}
+	{{with .Description}} {{.|prefixLines "// "}} {{end}}
 	type {{.GoType}} string
 	const (
-	{{ range $value := .Values -}}
+	{{ range $value := .Values}}
```
Comment (vektah) [inline codegen/templates/models.gotpl]: these -}} were removing unnecessary newlines

**Comment by vektah** [comment]
There are also field level descriptions that could be added

**Comment by andrioid** [comment]
I did briefly look into the FieldValue yesterday, but couldn't find Description to add.

I'll try to look into the newlines in the next few days.

**Comment by vektah** [comment]
Awesome, thanks for the PR

**Pull request merged**